compiler/fir/checkers/src/org/jetbrains/kotlin/fir/analysis/diagnostics/LightTreePositioningStrategies.kt

```diff
@@ -725,6 +735,13 @@ object LightTreePositioningStrategies {
                 }
             }
 
+            if (node.tokenType in namedIdentifierOwnerTokenTypes) {
```

Comment:
What are these changes for?

Reply:
This position strategy and its counterpart for PSI is not behaving in the same manner. Let me mention this in the commit message.